tracing/database.go

```diff
@@ -0,0 +1,141 @@
+package tracing
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/space307/go-utils/database"
+)
+
+type Database struct {
+	ExtDB *database.Database
```

Comment:
Is there need for is to be public?

Reply:
Yes, if I need use simple database.Database object

Reply:
Togda luche ispolzovat' getter

Reply:
I do not understand you @b13f

Reply:
@pilotuser TY DOLZHEN ISPOLZOVAT GETTER

Reply:
In my mind, we use this wrapper for traced execution only.
If we don't need tracing, and just want  database.Database, we should have it, not retrieve from this wrapper.

Reply:
ok, I added a getter.